eth2near/eth2near-block-relay-rs/Cargo.toml

```diff
@@ -32,6 +32,7 @@ hex = "*"
 toml = "0.5.9"
 atomic_refcell = "0.1.8"
 bitvec = "*"
+logger = { path = "../logger" }
```

Comment:
```suggestion
eth2near-logger = { path = "../logger" }
```

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/825/commits/4683dcd634d532d10f8c6f5b29479f1e224be26e